assets/js/modules/analytics/components/settings/SettingsForm.stories.js

```diff
@@ -79,6 +79,94 @@ WithoutModuleAccess.args = {
 	hasAnalyticsAccess: false,
 	hasAnalytics4Access: false,
 };
+WithoutModuleAccess.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			registry.dispatch( MODULES_ANALYTICS_4 ).setPropertyID( null );
+		};
```

Comment:
Is this necessary? If GA4 was connected but the user didnt have access, then the property ID should and is correctly displayed (but the select is disabled). I don't think we need to reset this. We could remove the whole decorator here.

Reply:
There should be two stories when the user doesn't have both the UA and GA4 access. I have implemented the one with the Activate GA4 switch disabled and the one you've proposed. They both are different scenarios.